linkerd/hatch.toml

```diff
@@ -0,0 +1,12 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+
+[envs.default.env-vars]
+DDEV_SKIP_GENERIC_TAGS_CHECK = "true"
+
+[envs.default]
```

Comment:
nit: let's put this before the matrix

Reply:
Done, will also applied that to the other PRs